src/main/java/org/jabref/gui/entryeditor/EntryEditor.java

```diff
@@ -153,71 +153,56 @@
      * Indicates that we are about to go to the next or previous entry
      */
     private final BooleanProperty movingToDifferentEntry = new SimpleBooleanProperty();
-    private final EntryType entryType;
+    private EntryType entryType;
     private SourceTab sourceTab;
+    private final BorderLayout layout;
+    private TypeLabel typeLabel;
 
-    public EntryEditor(JabRefFrame frame, BasePanel panel, BibEntry entry, String lastTabName) {
-        this.frame = frame;
+    public EntryEditor(BasePanel panel, BibEntry entry) {
```

Comment:
I would remove the `entry` from the constructor to make it clearer that the entry editor can be reused and is not specific to the entry.

Reply:
Done